src/components/reformed.js

```diff
@@ -91,9 +93,12 @@ const makeWrapper = WrappedComponent => {
 
       const value = get(name, this.state.model, '');
 
+      const checked = name === 'checkbox' ? value : undefined;
```

Comment:
Foreslår å endre til `bindInput(name, type)` hvor type kan være undefined. Så isFile kan endres til:
```js
      if (type === 'file') {
        return {
          type,
          name,
          onChange: this.bindToChangeEvent,
        };
      }
```

og ha en lignende sjekk for checkbox
```js
 const defaultProps = {
     name,
     onChange: this.bindToChangeEvent,
     onBlur: this.bindInputEvent,
     onFocus: this.bindInputEvent
};
if (type ==='checkbox') {
   return {...defaultProps, type,  checked: value}
}
return {...defaultProps, value}
```

Reply:
Du må også huske å endre der bindInput er brukt på fil inputs (se opplasting forms).